tests/unit/client/oauth2.test.ts

```diff
@@ -204,34 +245,37 @@ describe('OAuth2HttpClient', () => {
         },
       });
 
+    let mfaError;
+    try {
```

Comment:
(opinion / ignore) Just for cleanliness  as this is a large test, I would 
`await httpWithAuth.extraAuthMethods.authenticate(emailAuthData).catch(error => {
      mfaError = error;
 });`